ui/packages/atlasmap-core/src/services/mapping-expression.service.spec.ts

```diff
@@ -168,4 +174,56 @@ describe('MappingExpressionService', () => {
     const mapping1 = cfg.mappings!.mappings[1];
     expect(service.createMappingExpression(mapping1)).toContain('Concatenate');
   });
+
+  const mockstr = () => {
```

Comment:
Is there a special reason to have this as a function?

Reply:
removed mockstr - just added inline nameless fcns